src/main/resources/application.properties

```diff
@@ -1,7 +1,8 @@
-spring.datasource.url=jdbc:mysql://localhost:3306/book?Timezone=Europe/Kiev
-spring.datasource.username=root
-spring.datasource.password=1234
-spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
-
-spring.jpa.hibernate.ddl-auto=validate
-spring.jpa.show-sql=true
+spring.datasource.url=jdbc:mysql://localhost:3306/book?Timezone=Europe/Kiev
+spring.datasource.username=root
+spring.datasource.password=1234
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+spring.jpa.hibernate.ddl-auto=validate
+spring.jpa.show-sql=true
+jwt.expiration=100000
+jwt.secret=djflsagdkfjgkfdjnjgkdjdjsdjsjjdsijfdsls

```

Comment:
It's better to get sensitive properties from system properties and not pushing it to git.